Osmar Del Rosario y Sergio Arrocha/matriz1.c

```diff
@@ -0,0 +1,62 @@
+#include<stdio.h>
+
+void main(){
+	int c1,i,p, matriz1[10][10], matriz2[10][10],matriz3 [10][10],suma[10][10]; 
+	
+	printf("introduzca la cantidad de la matriz: ");
+	scanf("%d",&c1);
+	
+	for(i=0;i<c1;i++)
+	{	
+	for(p=0;p<c1;p++)
+		{
+		printf("ingrese los valores [%d,%d]:",i,p);		
+		 scanf("%d",&matriz1[i][p]);	
+}
+		printf("\n");
+}
+for(i=0;i<c1;i++)
+{
+	for(p=0;p<c1;p++)
+	{
+		printf("%d\t",matriz1[i][p]);
+		}	
+		printf("\n");
+}
+	printf("introduzca la segunda matriz: ");
+	scanf("%d",&c1);
+	
+	for(i=0;i<c1;i++)
+	{	
+	for(p=0;p<c1;p++)
+		{
+		printf("ingrese los valores [%d,%d]:",i,p);		
+		 scanf("%d",&matriz2[i][p]);	
+}
+		printf("\n");
+}
+for(i=0;i<c1;i++)
+{
+	for(p=0;p<c1;p++)
+	{
+		printf("%d\t",matriz2[i][p]);
+		}	
+		printf("\n");
+}
+
+printf("la mutiplicacion de matrices \n");
+    for(i=0;i<c1;i++)
+	{
+	for(p=0;p<c1;p++)
+		{
+				suma[i][p]= matriz1[i][p]+matriz2[i][p];

```

Comment:
La formula de multiplicacion de la matriz esta mal

Reply:
En la multiplicacion de matrices... si tenemos por ejemplo una matriz 2 x 2, siendo A y B dos matrices de la siguiente forma:
![54](https://user-images.githubusercontent.com/31838625/32155333-c363b8b6-bd04-11e7-9aee-ef6713a84758.gif)

Reply:
en tu logica, te falta un for, para conseguir la multiplicacion por la columna